Crash on API <26

I get a crash on API levels below 26 when changing route specific data for a route that has previous data. Relevant parts of stack trace:

```
Caused by: java.lang.NoSuchMethodError: No virtual method replace(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object; in class Ljava/util/HashMap; or its super classes (declaration of 'java.util.HashMap' appears in /system/framework/core-libart.jar)
        at org.rekotlinrouter.NavigationReducer$NavRed.setRouteSpecificData(NavigationReducer.kt:47)
        at org.rekotlinrouter.NavigationReducer$NavRed.handleAction(NavigationReducer.kt:25)
        at org.rekotlinrouter.NavigationReducer$NavRed.reduce(NavigationReducer.kt:62)
```

The problem is that `setRouteSpecificData()` calls [`HashMap.replace(K key, V value)`](https://developer.android.com/reference/java/util/HashMap.html#replace(K,%20V)), which wasn't added until API 26.

--- a/rekotlin-router/build.gradle
+++ b/rekotlin-router/build.gradle
@@ -9,7 +9,7 @@ apply plugin: 'maven-publish'
 
 
 group 'org.rekotlin-router'
-version '0.1.5'
+version '0.1.6'
 
 ext {
     junitPlugin_version = '1.0.0'

--- a/rekotlin-router/src/main/java/org/rekotlinrouter/NavigationReducer.kt
+++ b/rekotlin-router/src/main/java/org/rekotlinrouter/NavigationReducer.kt
@@ -43,11 +43,8 @@ class NavigationReducer {
         fun setRouteSpecificData(state: NavigationState, route: Route, data: Any): NavigationState {
             val routeString = FullRoute(route).routeString
 
-            if (state.routeSpecificState.containsKey(routeString)) {
-                state.routeSpecificState.replace(routeString, data)
-            } else {
-                state.routeSpecificState.put(routeString, data)
-            }
+            state.routeSpecificState[routeString] = data
+
             return state
         }
 

